packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -11,12 +11,53 @@ import {
     ISequencedDocumentMessage,
     MessageType,
 } from "@fluidframework/protocol-definitions";
-import { IDeltaManager } from "@fluidframework/container-definitions";
-import { MockDeltaManager } from "@fluidframework/test-runtime-utils";
-import { ScheduleManager } from "../containerRuntime";
+import { IContainerContext, IDeltaManager } from "@fluidframework/container-definitions";
+import { MockDeltaManager, MockQuorum } from "@fluidframework/test-runtime-utils";
+import { ContainerRuntime, ScheduleManager } from "../containerRuntime";
 
 describe("Runtime", () => {
     describe("Container Runtime", () => {
+        describe("ContainerRuntime", () => {
+            describe("orderSequentially", () => {
+                let containerRuntime: ContainerRuntime;
+                const mockContext: Partial<IContainerContext> = {
+                    deltaManager: new MockDeltaManager(),
+                    quorum: new MockQuorum(),
+                };
+
+                beforeEach(async () => {
+                    containerRuntime = await ContainerRuntime.load(
+                        mockContext as IContainerContext,
+                        [],
+                        undefined, // requestHandler
+                        {
+                            summaryOptions: {
+                                generateSummaries: false,
+                            },
+                        },
+                    );
+                });
+
+                it("Can't call flush() inside orderSequentially's callback", () => {
+                    assert.throws(
+                        () => containerRuntime.orderSequentially(() => containerRuntime.flush()),
+                        {
+                            message: "0x234",
```

Comment:
Tagging will break you (or rather you will break tagging). Tags are not really guaranteed, as reusing tag will change one of them (though I believe if they are in different files, that tagger will preserve the "older" callsite).
I think it's better not to rely on tags in UTs, just check that assert is thrown is enough.

Reply:
Was under the impression these were set in stone after tagging. Will revert this..